package.json

```diff
@@ -30,7 +30,7 @@
     "browser-sync": "^2.4.0",
     "browserify-istanbul": "^0.1.3",
     "browserify-shim": "^3.8.2",
-    "datatables": "1.10.9",
+    "datatables": "^1.10.9",
```

Comment:
We need to lock this down right? to version `1.10.11` otherwise we potentially will have the same problem in the future if the owners release a breaking change without using a breaking change `semver` version?